libs/components/modals/src/lib/modules/modal/modal.component.html

```diff
@@ -20,7 +20,7 @@
   >
     <div
       class="sky-modal-header"
-      [hidden]="!headerContent || !headerContent.children || headerContent.children.length &lt; 1"
```

Comment:
Since we're adding conditional content into the header section, this was throwing a changed-after-checked error and had to be refactored.